### DISCUSSION TITLE
How to not auto start services when start computer?

### DISCUSSION THREAD

**Discussion by kittiu** [rtCamp/Frappe-Manager#151] [Q&A] [answered]
I am using ubuntu, and when I start the computer, all the sites started. How to prevent that, as it seem to consume CPU a lot.

**Comment by Xieyt** [comment]
This was fixed in `0.12.0` update. Only your `db` and `nginx-proxy` is set to auto start.

You can use this command to turn this off.
```
find ~/frappe -type f -name 'docker-compose*.yml' -exec sed -i '/restart: always/d' {} +
```

The above finds all the `docker-compose*.yml` in `~/frappe` directory and then remove `restart: always` from the compose file.

**Reply by kittiu** [reply, marked as answer]
Thanks!
